core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -83,7 +83,7 @@ class AdminClient(val time: Time,
   }
 
   private def findAllBrokers(): List[Node] = {
-    val request = new MetadataRequest(List[String]())
+    val request = new MetadataRequest(List[String](), true)
```

Comment:
I think It depends on the choice in earlier comments here: https://github.com/apache/kafka/pull/1095#discussion_r60764750